Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -0,0 +1,20 @@
+- type: entity
+  name: stick of ganja butter
+  parent: FoodBakingBase
+  id: FoodGanjaButter
+  description: Add this to your favorite baked goods for an irie time.
+  components:
+  - type: Sprite
+    state: butter
+    color: "#82C36E"
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 90
+        reagents:
+        - ReagentId: Butter
+          Quantity: 10
+        - ReagentId: THCOil
```

Comment:
Do I understand this correctly that you end up doubling the THC oil available when you make this food? 8 leaves, each leaf has 5 oil, and at the end, you have 80, instead of 40.

Reply:
Balanced to 9 leaves->1 butter=45THC, 2 sticks per batch of 6 brownies (90THC->15THC)